src/components/filter/filter-component/Ratings-filter.jsx

```diff
@@ -74,6 +77,7 @@ const RatingsFilter = ({ filteredApplyItemState, filteredApplyItemDispatch}) =>
                                 type="radio"
                                 id="rating-1"
                                 className="input-radio"
+                                checked ={filteredApplyItemState.ratings === 1 ? true : false}
```

Comment:
In all the radio buttons, you can directly write checked ={filteredApplyItemState.ratings === 1}

like this